### DISCUSSION TITLE
please allow data from vulnerabilities to be exported to csv like you can do for risk

### DISCUSSION THREAD

**Discussion by ngamboa** [community/community#113324] [Code Security]
### Select Topic Area

Product Feedback

### Body

please include an export to csv option for the dependabot, code and secret alerts, not everyone works directly out of github and for risk analysis we need to easily get this data out of the system, having to do painful api reports is not a useful option for non-developers.

**Comment by kellyarwine** [comment]
Thank you for your feedback, @ngamboa! We're excited to share that a public beta for this feature is planned for the near term (April 2024).  You'll have the capability to export alerts from the overview dashboard directly to CSV. Post-launch, after reviewing user feedback, we'll move to shipping an API (with summary and detail data). I would guesstimate the APIs will roll out within the next 3-4 months. You can follow along [here](https://github.blog/changelog/label/security-and-compliance/) for all announcements.
